### DISCUSSION TITLE
How to move focus to other group on dual monitor setup

### DISCUSSION THREAD

**Discussion by chr-sa** [qtile/qtile#2796] [Q&A] [answered]
I have a dual monitor (screen) setup and I would like to move focus from one screen to the other using the hjkl keys.

For example:
The screens are side by side and I am focused on the rightmost window on the left screen. Now I want to be able to press mod + "l" to be able to move focus to the leftmost window on the right screen.

Would this be somehow possible? I did sadly not find anything about how to achieve that but it seems so simple that I am pretty sure I am just missing something obvious. Any help would be appreciated, thanks!

**Comment by m-col** [comment, marked as answer]
I have some functions to do this that work nicely. I'm adding them to the examples repo: https://github.com/qtile/qtile-examples/pull/59. Specifically the ones you want I have called traverse: https://github.com/qtile/qtile-examples/pull/59/commits/e389beb102ec013a2b6874b16717b9596d85f13d#diff-e82271f9bd3d9e83b0dd00dd9f27b3c66151bb9421878a637aab1e1526b045fe

**Reply by chr-sa** [reply]
I just tried it out, works flawlessly. Thank you very much for this!

**Reply by agamm** [reply]
Just a note: I had to create a new file named `traverse.py` next to my `config.py` and import it.
